rasterio/rio/helpers.py

```diff
@@ -64,11 +67,29 @@ def write_features(
         fobj.write('\n')
 
 
-def resolve_inout(input=None, output=None, files=None):
+def resolve_inout(input=None, output=None, files=None, force_overwrite=False):
```

Comment:
Here's the crux of the PR, @perrygeo @geowurster: a new keyword argument for `resolve_inout()` and use of it throughout the CLI.